packages/atproto_client/models/string_formats.py

```diff
@@ -54,6 +54,19 @@
 )
 
 
+class _NamedValidator:
+    """Decorator to add a __str__ attribute to a validation function."""
+
+    def __init__(self, validate_fn: Callable[..., str]) -> None:
+        self.validate_fn = validate_fn
+
+    def __call__(self, v: str, info: ValidationInfo) -> str:
+        return self.validate_fn(v, info)
+
+    def __str__(self) -> str:
+        return f'Validated by: {self.validate_fn.__name__} (only when `strict_string_format=True`)'
```

Comment:
doh! i tried almost exactly this but __repr__ instead and it broke the docstrings in this file